src/Simulation/EntryPointDriver/Azure/Azure.cs

```diff
@@ -432,7 +432,9 @@ private static void LogIfVerbose(GenerateAzurePayloadSettings settings, string m
             {
                 NoOpQirSubmitter.Target => new NoOpQirSubmitter(),
                 NoOpSubmitter.Target => new NoOpSubmitter(),
-                _ => SubmitterFactory.QirSubmitter(settings.Target, settings.CreateWorkspace(), settings.Storage)
+                _ => (!string.IsNullOrWhiteSpace(settings.TargetCapability)) 
```

Comment:
This is slightly more nuanced than just checking whether target capability is null or empty. Submitters for some targets require target capability to be non-null and non-empty, but others don't have that requirement. I suggest adding a property to the `SubmitterInfo` class: `public bool RequiresTargetCapability`. This can be used inside the `SubmitterFactory.QirSubmitter` method to check the value of `settings.TargetCapability` and act accordingly.

@kuzminrobin please let me know if this makes sense.